controllers/users.js

```diff
@@ -8,6 +8,24 @@ const imageService = require('../services/imageService')
  * @param res {Object} - Express response object
  */
 
+
+const verifyUser = async (req, res) => {
+  try {
+    const userId = req.userData.id;
+    await userQuery.addOrUpdate(req.userData.identityStatus, userId);
+    let fetchData = {
+      body: JSON.stringify({
+        username: req.userData.username,
+        identityURL: req.userData.identityURL,
+      }),
+    };
```

Comment:
Add method and headers in the options

Reply:
Done updated the method and headers type